modules/i3s/src/lib/utils/customize-сolors.ts

```diff
@@ -1,12 +1,27 @@
-import type {MeshAttribute} from '@loaders.gl/schema';
-import type {COLOR, I3STileOptions, I3STilesetOptions} from '../../types';
+import type {MeshAttribute, TypedArray} from '@loaders.gl/schema';
+import type {AttributeStorageInfo, COLOR, Field} from '../../types';
 
 import {load} from '@loaders.gl/core';
 import {getAttributeValueType, I3SAttributeLoader} from '../../i3s-attribute-loader';
 import {I3SLoaderOptions} from '../../i3s-loader';
 import {getUrlWithToken} from './url-utils';
 import {I3STileAttributes} from '../parsers/parse-i3s-attribute';
 
+type ColorsByAttribute = {
+  /** Feature attribute name */
+  attributeName: string;
+  /** Minimum attribute value */
+  minValue: number;
+  /** Maximum attribute value */
+  maxValue: number;
+  /** Minimum color. 3DObject will be colorized with gradient from `minColor to `maxColor` */
+  minColor: [number, number, number, number];
+  /** Maximum color. 3DObject will be colorized with gradient from `minColor to `maxColor` */
+  maxColor: [number, number, number, number];
+  /** Colorization mode. `replace` - replace vertex colors with a new colors, `multiply` - multiply vertex colors with new colors */
+  mode: string;
+};
+
 /**
  * Modify vertex colors array to visualize 3D objects in a attribute driven way
```

Comment:
It no longer modifies anything, right? Can we update the jsdoc?

Reply:
Updated